backend/plugin/db/pg/dump.go

```diff
@@ -20,6 +20,10 @@ import (
 	storepb "github.com/bytebase/bytebase/proto/generated-go/store"
 )
 
+var (
+	pgdumpRootCertPath = "/usr/local/share/ca-certificates/ca-certificates.crt"
```

Comment:
This could vary based on the OS.

Reply:
https://go.dev/src/crypto/x509/root_linux.go

Reply:
https://pkg.go.dev/crypto/x509#SystemCertPool

Should this work?

Reply:
It depends on whether we need to consider the self-signed ca.
### MacOS
1. The system certs are stored in the Keychain utility, and access them out of Keychain is not recommended. https://superuser.com/questions/992167/where-are-digital-certificates-physically-stored-on-a-mac-os-x-machine
2. The user self-signed ca should be add in Keychain utility and mark it is trusted like this:
![CleanShot 2024-08-05 at 10 30 07@2x](https://github.com/user-attachments/assets/a07990d3-ef34-4da5-9e36-18370f49c735)

### Debian(our docker image based)
1. The system certs are mostly stored in the `/etc/ssl/certs/ca-certificates.crt`, and some are stored in the `/usr/share/ca-certificates/xxx.crt`.
![CleanShot 2024-08-05 at 10 39 21@2x](https://github.com/user-attachments/assets/efc60c69-eae2-44a1-a9c3-1e084ec7924b)
2. The user self-signed ca should be added in `/usr/share/ca-certificates/xxx.crt`. https://www.baeldung.com/linux/add-self-signed-certificate-trusted-list


### pg_dump restrictions
1. Use cert whose path is stored in `PGSSLROOTCERT` environment variable to verify server side certificate.
2. The `PGSSLROOTCERT` should store the path of **one cert file**.

Reply:
Add more comments with details in the code?